demisto_sdk/commands/common/hook_validations/id.py

```diff
@@ -397,3 +400,37 @@ def is_file_valid_in_set(self, file_path, file_type, ignored_errors=None):
                 is_valid = self._are_playbook_entities_versions_valid(playbook_data, file_path)
 
         return is_valid
+
+    def _is_pack_display_name_already_exist(self, pack_metadata_data):
+        """Check if the pack display name already exists in our repo
+        Args:
+            pack_metadata_data (dict): Dictionary that holds the extracted details from the given metadata file.
+        Returns:
+            bool. Whether the metadata file is valid or not.
+        """
+        new_pack_folder_name = list(pack_metadata_data.keys())[0]
+        new_pack_name = pack_metadata_data[new_pack_folder_name]['name']
+        for pack_folder_name, pack_data in self.packs_set.items():
+            if new_pack_name == pack_data['name'] and new_pack_folder_name != pack_folder_name:
```

Comment:
Can you explain the second condition here in details please?

Reply:
https://github.com/demisto/demisto-sdk/pull/1122#discussion_r577585476